examples/app_commands/Info.py

```diff
@@ -0,0 +1,32 @@
+import discord
+from discord.ext import commands
+# imports
+
+intents = discord.Intents(
+            guilds=True,
+            members=True,
+            messages=True,
+            )
+
+bot = commands.Bot(command_prefix=".", description="An example to showcase how to extract info about users",intents=intents)
+
+@bot.slash_command(name="userinfo", description="gets the info of a user")
+async def info(ctx, user: discord.Member = None):
+    user = user or ctx.author #if no user is provided it'll use the the author of the message
+    e = discord.Embed()
+    e.set_author(name=user.name)
+    e.add_field(name='ID', value=user.id, inline=False) # user ID
```

Comment:
```suggestion
    e.add_field(name="ID", value=user.id, inline=False) # user ID
```
Keeps consistency with the double quoted strings the file sets precedent with.